docs/BreakingChanges/141 WPF TreeViewItem must be used within a TreeView.md

```diff
@@ -0,0 +1,27 @@
+## ID: WPF TreeViewItem must be used within a TreeView
+
+### Scope
+Major
```

Comment:
I would probably call this minor because ApiPort will only be able to detect TreeViewItem usage, and most of those will be within TreeViews. So, I think major is too strong of a scope.